capitulo 6/7-Pessoas2.py

```diff
@@ -0,0 +1,10 @@
+person_0 = {'first_name':'Derick','last_name':'Pereira','age':14,'city':'Rio de Janeiro'}
+person_1 = {'first_name':'Dylan','last_name':'Pereira','age':18,'city':'Rio de Janeiro'}
+person_2 = {'first_name':'Luke','last_name':'Dias','age':24,'city':'Rio de Janeiro'}
+
+people = (person_0,person_1,person_2)
```

Comment:
Detalhe, people aqui está como uma tupla, enquanto o livro pede pra ser uma lista.
Está funcionando pois a tupla é para leitura, que é o que você está fazendo aqui, porém caso quisesse adicionar mais um elemento à lista, não seria possível.
```suggestion
people = [person_0,person_1,person_2]
```